src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -25,4 +26,20 @@ object NumericUtils {
    */
   def applyTolerance(values: Seq[Double]): Seq[Double] =
     values map applyTolerance
+
+  /**
+    * Rounds 7th decimal place so that number has 6 decimal places.
+    * @param d A Double value
+    * @return Value with 6 decimal places
+    */
+  def roundToTolerance(d: Double): Double =
```

Comment:
How about we make it so that the number of decimal places is a parameter, and we use 6 as the default value?